Jenkinsfile

```diff
@@ -9,17 +9,19 @@ pipeline {
         skipDefaultCheckout()
         buildDiscarder(logRotator(artifactDaysToKeepStr: '1', artifactNumToKeepStr: '1', daysToKeepStr: '5', numToKeepStr: '10'))
     }
+    parameters {
+        booleanParam(name: 'UNIT_TESTS', defaultValue: true, description: 'Run Unit tests')
+        booleanParam(name: 'MOCK_MVC_TESTS', defaultValue: true, description: 'Run Mock MVC tests')
+        booleanParam(name: 'INTEGRATION_TESTS', defaultValue: true, description: 'Run Integration tests')
+        booleanParam(name: 'BUILD_TO_STAGE', defaultValue: false, description: 'Publish to build artifactory')
```

Comment:
can we change the name of `BUILD_TO_STAGE`? too confusing when it means something different in old jenkins